### DISCUSSION TITLE
Default Account

### DISCUSSION THREAD

**Discussion by quocduong** [JordanKnott/taskcafe#144] [Q&A]
I used docker compose to build an offline (no email activate). How I can setup a default account as administrator?

**Comment by Unrepentant-Atheist** [comment]
Same here, I did `docker-compose -p taskcafe up -d` and went to localhost:3333 and there was no setup, just ready to be logged in:

![image](https://user-images.githubusercontent.com/3938423/160346843-5b9b8ff7-a621-4ead-845d-86d6f4b70bde.png)

Clicking register does nothing.

**Reply by Jsharkc** [reply]
type localhost:3333/register，and register
